localization/english/religion/wc_religion_l_english.yml

```
@@ -5,4 +5,502 @@
+ 
+ earth_group:0 "Geoism"
+ earth_group_adj:0 "Geo"
+ earth_group_adherent:0 "Geoist"
+ earth_group_adherent_plural:0 "Geoists"
+ earth_group_desc:0 "Among all the elements, the Geoists regard earth as the main force and worship earth elementals as deities."
+ 
+ therazane_worship:0 "Therazanism"
+ therazane_worship_adj:0 "Therazanic"
+ therazane_worship_adherent:0 "Therazanist"
+ therazane_worship_adherent_plural:0 "Therazanists"
+ therazane_worship_desc:0 "$therazane_worship_adherent_plural$ worship one of the Elemental Lords, Therazane the Stonemother, as their main god. Other earth elementals are seen as minor deities."
+ 
+ theradras_worship:0 "Theradrasism"
+ theradras_worship_adj:0 "Theradrasic"
+ theradras_worship_adherent:0 "Theradrasist"
+ theradras_worship_adherent_plural:0 "Theradrasists"
+ theradras_worship_desc:0 "Princess Theradras was an enormous earth elemental and a daughter of Therazane, the Stonemother. After mating with Zaetar, son of Cenarius, Theradras became the mother of the first centaur. The centaurs, seeing the disgust in their father's eyes, brutally killed Zaetar. And to this day, Theradras is the patron of her progeny."
+ 
+ fel_group:0 "Fel"
+ fel_group_adj:0 "Fel"
+ fel_group_adherent:0 "Fel Follower"
+ fel_group_adherent_plural:0 "Fel Followers"
+ fel_group_desc:0 "$fel_group_adherent_plural$ belive Chaos is the natural order of things. They master Fel magic, created as a result of the mutual destruction of Light and Void."
+ 
+ burning_legion_religion:0 "Sargerism"
+ burning_legion_religion_adj:0 "Sargerite"
+ burning_legion_religion_adherent:0 "Sargerite"
+ burning_legion_religion_adherent_plural:0 "Sargerites"
+ burning_legion_religion_desc:0 "Bent on destroying all life in the universe, Sargeras is seen as one of the greatest threats to creation, and in a smaller sense, Azeroth. At his beck and call during his burning crusade across the Great Beyond is a loyal, fel-infused legion of demons, the Burning Legion, ready to carry out the destruction Sageras so very much desires. Followers of the Dark Titan are only capable of cruelty and deceit, as they chase any chance to bring about the extinguishment of life."
+ 
+ illidari:0 "Vindicatorism"
+ illidari_adj:0 "Vindicatory"
+ illidari_adherent:0 "Vindicatorist"
+ illidari_adherent_plural:0 "Vindicatorists"
+ illidari_desc:0 "The powers of fel can be found tempting to all living beings in the universe, but there are those who use it as a tool to fight the evils that lurk throughout creation, rather than for personal gain or destruction. Though these individuals can often be found guilty of similar crimes to that of power-hungry warlocks and demons, they see any suffering inflicted as means to a greater end, an end greater than themselves and their sacrifices. Though more realistically benevolent than most demons and warlocks, society will seek to reject them and their cruel, even if effective, methods nonetheless."
```

Removed even in the last sentence and found in the first, as they are unnecessary. Changed `to that of` to `as those of` as I think it makes more sense.
```suggestion
 illidari_desc:0 "The powers of fel can be tempting to all living beings in the universe, but there are those who use it as a tool to fight the evils that lurk throughout creation, rather than for personal gain or destruction. Though these individuals can often be found guilty of similar crimes as those of power-hungry warlocks and demons, they see any suffering inflicted as means to a greater end, an end greater than themselves and their sacrifices. Though more realistically benevolent than most demons and warlocks, society will seek to reject them and their cruel, if effective, methods nonetheless."
```